test/common/upstream/eds_test.cc

```diff
@@ -30,7 +30,10 @@ namespace {
 
 class EdsTest : public testing::Test {
 protected:
-  EdsTest() : api_(Api::createApiForTest(stats_)) { resetCluster(); }
+  EdsTest() : api_(Api::createApiForTest(stats_)) {
+    resetCluster();
+    fare_cluster_load_assignment_.set_cluster_name("fare");
```

Comment:
Why this change?

Reply:
Most of the tests use this same "fare" one, and declaring locally uses an extra 3 lines. I thought it was borderline even when I first changed it, so I'm fine with keeping them local.